cli/src/main.rs

```diff
@@ -20,15 +22,17 @@ enum Commands {
     ConsumeNotes(ConsumeNotesCmd),
     Init(InitCmd),
     Register(RegisterCmd),
+    Connect(ConnectCmd),
+    Stats(StatsCmd)
 }
 
 #[tokio::main]
 async fn main() {
     let args = Args::parse();
-
+    let ws_config_path = std::path::PathBuf::from("ws_config.json");
```

Comment:
Are we assuming user to have a `ws_config.json` with `url` field empty in it for the time being. Untill the game is not initialized.

Reply:
No if ws_config.json isn't present it'll create one